tests/test_nvtx_transform.py

```diff
@@ -116,33 +119,92 @@ def test_nvtx_transfroms_array(self, input):
 
     @parameterized.expand([TEST_CASE_DICT_0, TEST_CASE_DICT_1])
     @unittest.skipUnless(has_nvtx, "CUDA is required for NVTX!")
-    def test_nvtx_transfromsd(self, input):
+    def test_nvtx_transfroms_dict(self, input):
+        # with prob == 0.0
         transforms = Compose(
             [
-                RandMarkD("Mark: Transform Starts!"),
+                RandMarkD("Mark: Transforms (p=0) Start!"),
                 RandRangePushD("Range: RandFlipD"),
                 RandFlipD(keys="image", prob=0.0),
                 RandRangePopD(),
                 RangePushD("Range: ToTensorD"),
                 ToTensorD(keys=("image")),
                 RangePopD(),
-                MarkD("Mark: Transform Ends!"),
+                MarkD("Mark: Transforms (p=0) End!"),
             ]
         )
         output = transforms(input)
         self.assertIsInstance(output["image"], torch.Tensor)
         np.testing.assert_array_equal(input["image"], output["image"])
-
+        # with prob == 1.0
         transforms = Compose(
             [
-                RandMarkD("Mark: Transform Starts!"),
+                RandMarkD("Mark: Transforms (p=1) Start!"),
                 RandRangePushD("Range: RandFlipD"),
                 RandFlipD(keys="image", prob=1.0),
                 RandRangePopD(),
                 RangePushD("Range: ToTensorD"),
                 ToTensorD(keys=("image")),
                 RangePopD(),
-                MarkD("Mark: Transform Ends!"),
+                MarkD("Mark: Transforms (p=1) End!"),
+            ]
+        )
+        output = transforms(input)
+        self.assertIsInstance(output["image"], torch.Tensor)
+        np.testing.assert_array_equal(input["image"], Flip()(output["image"].numpy()))
+
+    # ---------------------
+
+    @parameterized.expand([TEST_CASE_ARRAY_0, TEST_CASE_ARRAY_1])
+    @unittest.skipUnless(has_nvtx, "CUDA is required for NVTX!")
+    def test_nvtx_range_array(self, input):
+        # with prob == 0.0
+        transforms = Compose(
+            [
+                RandMark("Mark: Transforms (p=0) Start!"),
+                Range(RandFlip(prob=0.0)),
+                Range(ToTensor()),
+                Mark("Mark: Transforms (p=0) End!"),
+            ]
+        )
+        output = transforms(input)
+        self.assertIsInstance(output, torch.Tensor)
+        np.testing.assert_array_equal(input, output)
+        # with prob == 1.0
+        transforms = Compose(
+            [
+                RandMark("Mark: Transforms (p=1) Start!"),
+                Range(RandFlip(prob=1.0)),
```

Comment:
Could you please also add test cases for `RandRange`?

Thanks.

Reply:
Thanks for the comment. I added `RandRange` and `RandRangeD`.